prime-router/metadata/fhir_mapping/hl7/resource/Patient.yml

```diff
@@ -214,7 +214,7 @@ identifier_8:
     display: "Patient's Medicare number"
 
 name:
-  valueOf: datatype/HumanName
+  valueOf: datatypes/FN/HumanName
```

Comment:
We only should be updating this in one spot just so that we can write tests against it